Open Judge System/Data/OJS.Data/OjsDbContext.cs

```diff
@@ -176,6 +176,7 @@ private void ApplyAuditInfoRules()
                     if (!entity.PreserveCreatedOn)
                     {
                         entity.CreatedOn = DateTime.Now;
+                        entity.ModifiedOn = null;
```

Comment:
This is wierd. !PreserveCreatedOn, yet CreatenOn is preserved

Reply:
new value is assigned to CreatedOn, the old value is not preserved